### ISSUE TITLE
Windows Playbook Checker PR check failing signature verification steps in playbook

### ISSUE THREAD

**Issue by Haroon-Khel** [adoptium/infrastructure#3067]
ref https://github.com/adoptium/infrastructure/pull/3018

The changes made in that PR concern adding signature verification steps to verify downloads in the Windows playbook. Cygwin's own GnuPG is used which works in a cmd session (over remote desktop) and the ansible tasks work when executed from my workstation, but the PR check job is having trouble with them

```
TASK [ANT : Import GPG Key] ****************************************************
fatal: [127.0.0.1]: FAILED! => 
{"changed": true, "cmd": "gpg --keyserver keyserver.ubuntu.com --recv-keys \"CE8075A251547BEE249BC151A2115AE15F6B8B72\"", "delta": "0:00:00.072935", "end": "2023-04-05 12:25:35.885799", 
"msg": "non-zero return code", "rc": 2, "start": "2023-04-05 12:25:35.812864", "stderr": "gpg: fatal: can't create directory `/home/runneradmin/.gnupg': No such file or directory\nsecmem usage: 0/0 bytes in 0/0 blocks of pool 0/65536\n", 
"stderr_lines": ["gpg: fatal: can't create directory `/home/runneradmin/.gnupg': No such file or directory", "secmem usage: 0/0 bytes in 0/0 blocks of pool 0/65536"], "stdout": "", "stdout_lines": []}
```

**Comment by Haroon-Khel** [comment]
Blocks https://github.com/adoptium/infrastructure/issues/2908

**Comment by steelhead31** [comment]
Fixed as part of #3018
